assets/css/translation.css

```diff
@@ -126,3 +126,67 @@ html[dir='rtl'] #srcLanguages {
     padding: 0 15px;
     position: relative;
 }
+
+#localeSelectContainer {
+     width: 35%;
+}
+
+#primaryLocaleGlobe {
```

Comment:
These top two rules should probably go in general, they aren't related to translation.